.github/workflows/test.yml

```
@@ -133,6 +132,7 @@ jobs:
           - '10.0'
           - '8'
           - '8.6'
+        context: [ 'patch', '' ]
```

Nice!

```suggestion
        contextManager: [ 'patch', '' ]
```

Can we call this `contextManager` to match the var in https://github.com/elastic/apm-agent-nodejs/pull/3126/files#diff-a8e31e174e3ddce38cb2351c978bb5d63bd46c41235a13c9dc12763e26afef05R63 doing the same thing?

---

See 413f021b630f33dddbd520483ebaea22662f3d35